client/src/reducers/grantsMetadata.test.ts

```diff
@@ -30,8 +37,15 @@ describe("newGrant reducer", () => {
       wallet: "0x000",
       receivedFunding: false,
     };
-    store.dispatch(grantMetadataFetched(grantMetadata));
-    expect(store.getState().grants[1]).toEqual({
+    store.dispatch(
+      grantMetadataFetched({
+        ...grantMetadata,
+        uri: grantMetadata.uri,
```

Comment:
@schultztimothy I think we can remove this line since it's overriding uri with the same value